code/modules/mob/living/living_defense.dm

```diff
@@ -223,32 +223,46 @@
 	log_combat(M, src, "attacked")
 	return TRUE
 
+/mob/living/attack_hand(mob/living/carbon/human/user)
+	. = ..()
+	if (user.apply_martial_art(src))
+		return TRUE
 
 /mob/living/attack_paw(mob/living/carbon/monkey/M)
 	if(isturf(loc) && istype(loc.loc, /area/start))
 		to_chat(M, "No attacking people at spawn, you jackass.")
 		return FALSE
 
-	if (M.a_intent == INTENT_HARM)
-		if(HAS_TRAIT(M, TRAIT_PACIFISM))
-			to_chat(M, "<span class='warning'>You don't want to hurt anyone!</span>")
-			return FALSE
+	if (M.apply_martial_art(src))
+		return TRUE
 
-		if(M.is_muzzled() || M.is_mouth_covered(FALSE, TRUE))
-			to_chat(M, "<span class='warning'>You can't bite with your mouth covered!</span>")
-			return FALSE
-		M.do_attack_animation(src, ATTACK_EFFECT_BITE)
-		if (prob(75))
-			log_combat(M, src, "attacked")
-			playsound(loc, 'sound/weapons/bite.ogg', 50, TRUE, -1)
-			visible_message("<span class='danger'>[M.name] bites [src]!</span>", \
-							"<span class='userdanger'>[M.name] bites you!</span>", "<span class='hear'>You hear a chomp!</span>", COMBAT_MESSAGE_RANGE, M)
-			to_chat(M, "<span class='danger'>You bite [src]!</span>")
+	switch (M.a_intent)
+		if (INTENT_HARM)
+			if(HAS_TRAIT(M, TRAIT_PACIFISM))
+				to_chat(M, "<span class='warning'>You don't want to hurt anyone!</span>")
+				return FALSE
+
+			if(M.is_muzzled() || M.is_mouth_covered(FALSE, TRUE))
+				to_chat(M, "<span class='warning'>You can't bite with your mouth covered!</span>")
+				return FALSE
+			M.do_attack_animation(src, ATTACK_EFFECT_BITE)
+			if (prob(75))
+				log_combat(M, src, "attacked")
+				playsound(loc, 'sound/weapons/bite.ogg', 50, TRUE, -1)
+				visible_message("<span class='danger'>[M.name] bites [src]!</span>", \
+								"<span class='userdanger'>[M.name] bites you!</span>", "<span class='hear'>You hear a chomp!</span>", COMBAT_MESSAGE_RANGE, M)
+				to_chat(M, "<span class='danger'>You bite [src]!</span>")
```

Comment:
... where's the damage for the bite? is it elsewhere, or do monkeys currently not deal damage to generic living mobs with their bites?

Reply:
Yeah everything just overrides the proc

Reply:
would it be a problem to add 1d3 damage to this proc, then, in case something DOESN'T override it?

Reply:
I mean it would give bonus damage